### PR TITLE
CKV_K8S_100 k8s api server tls cert and key

### PR DESCRIPTION
**Pull request by SteveVaknin**
Ensure that the --tls-cert-file and --tls-private-key-file arguments are set as appropriate
Id = CKV_K8S_100

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: checkov/kubernetes/checks/ApiServerTlsCertAndKey.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    def scan_spec_conf(self, conf):
+        if "command" in conf and conf["command"] is not None:
+            if "kube-apiserver" in conf["command"]:
+                hasCertCommand = False
+                hasKeyCommand = False
+                for command in conf["command"]:
+                    if command.startswith("--tls-cert-file"):
+                        hasCertCommand = True
+                    elif command.startswith("--tls-private-key-file"):
+                        hasKeyCommand = True
```
Comment (nimrodkor) [inline checkov/kubernetes/checks/ApiServerTlsCertAndKey.py:26]: @SteveVaknin Can be shortened into:
```python
if command.startswith("--tls-cert-file") or command.startswith("--tls-private-key-file"):
    hasCertCommand = True
```

File: tests/kubernetes/checks/test_ApiServerTlsCertAndKey.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+
+        test_files_dir = current_dir + "/example_ApiServerTlsCertAndKey"
+        report = runner.run(root_folder=test_files_dir, runner_filter=RunnerFilter(checks=[check.id]))
+        summary = report.get_summary()
+
+        self.assertEqual(summary['passed'], 1)
+        self.assertEqual(summary['failed'], 2)
+        self.assertEqual(summary['skipped'], 0)
+        self.assertEqual(summary['parsing_errors'], 0)
```
Comment (nimrodkor) [inline tests/kubernetes/checks/test_ApiServerTlsCertAndKey.py:22]: Please check specific resources are as expected

File: checkov/kubernetes/checks/ApiServerTlsCertAndKey.py (hunk 1 above)
Comment (SteveVaknin) [reply checkov/kubernetes/checks/ApiServerTlsCertAndKey.py:26]: @nimrodkor I need to make sure it has both

Comment (nimrodkor) [reply checkov/kubernetes/checks/ApiServerTlsCertAndKey.py:26]: @SteveVaknin Same comments as #41

**Review by nimrodkor — APPROVED** [review]

**Pull request merged**